src/lib/ga.tsx

```diff
@@ -39,11 +40,12 @@ export const event = ({
 export const GAScripts = () => {
   return (
     <>
-      <script
+      <Script
+        strategy="afterInteractive"
         async
         src={`https://www.googletagmanager.com/gtag/js?id=${gaTrackingId}`}
       />
-      <script
+      <Script
```

Comment:
missing strategy here i think

```suggestion
      <Script
        strategy="afterInteractive"
```